heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronExecutorTask.java

```diff
@@ -133,6 +133,7 @@ public HeronExecutorTask(final REEFFileNames fileNames,
         env,
         topologyJar,
         topologyDefFile,
+        false,
```

Comment:
no verbose option in [HeronTaskConfiguration](https://github.com/twitter/heron/blob/e68943d422ea35f54c46fc8944c1c83ef6440d78/heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronTaskConfiguration.java)